Formula/yelp-tools.rb

```diff
@@ -15,30 +16,47 @@ class YelpTools < Formula
   end
 
   depends_on "gettext" => :build
-  depends_on "intltool" => :build
   depends_on "itstool" => :build
   depends_on "libxml2" => :build
-  depends_on "libxslt" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
   depends_on "gtk+3"
+  depends_on "python@3.9"
+
+  uses_from_macos "libxslt" => :build
```

Comment:
Wasn't sure whether better to keep Homebrew's version or use macOS.
Switched based on common choice in formulae.